libs/cyassl/Makefile

```diff
@@ -1,61 +0,0 @@
-#
```

Comment:
Do you know why we had this forked?

Reply:
It's not a fork but a backport.
It looks like we were using an older version than Chaos Calmer so Kostko backported it and it was never dropped later.
Now it is not used for anything as CyaSSL was renamed to WolfSSL a while ago in OpenWrt and is crucial to hostapd.

Reply:
Cool, thanks.